libs/ui/design-system/src/lib/switch.tsx

```diff
@@ -1 +1,39 @@
-export { Switch } from '@nmit-coursition/ui/primitives/switch'
+import { cva, type VariantProps } from 'class-variance-authority'
+
+import { Label } from '@nmit-coursition/ui/primitives'
+import * as SwitchPrimitive from '@nmit-coursition/ui/primitives/switch'
+import { cn } from '@nmit-coursition/ui/utils'
+
+/** todo: import util from `typescript` file when mixins architecure is merged  */
+type OverrideProps<T, V> = V & Omit<T, keyof V>
+
+const switchVariants = cva('', {
+  variants: {
+    variant: {
+      /**
+       * OFF: bg-#4A044E, thumb-#FEF3C7
+       * ON: bg-[#FEF3C7], thumb-[#4A044E]
+       */
+      default: '',
+      disabled: '',
+      outlined: '', // default variant with an outline
```

Comment:
**question:** Do we have any outlined variant defined? I don't see such in Figma.

Reply:
This is what i mean
![error](https://github.com/user-attachments/assets/61a218dc-b92f-4548-b268-fdf7c96d6f5b)

Reply:
That's focus state. It's not a variant. :) You can see it in the right panel of properties.

![image](https://github.com/user-attachments/assets/92c5be29-2d9c-4d9a-9edb-2f854bd6855f)